dev/lerna.ts

```diff
@@ -0,0 +1,787 @@
+const SPLIT_CHAR = "END";
+
+const getPackages: Fig.Generator = {
+  script: "lerna ls",
+  postProcess: (output) =>
+    output.split("\n").map((packageName) => ({
+      name: packageName,
```

Comment:
So this would become just:
```suggestion
      name,
```